store/tikv/client_test.go

```diff
@@ -188,9 +208,18 @@ func (s *testClientSuite) TestCollapseResolveLock(c *C) {
 
 	// Mixed
 	wg.Add(1)
-	go client.SendRequest(ctx, "", resolveLockReq, time.Second)
-	go client.SendRequest(ctx, "", resolveLockLiteReq, time.Second)
-	go client.SendRequest(ctx, "", batchResolveLockReq, time.Second)
+	go func() {
+		_, err := client.SendRequest(ctx, "", resolveLockReq, time.Second)
+		c.Assert(err, IsNil)
```

Comment:
`Assert` will only affect the current goroutine. I think we'd better handle the error outside of this goroutine.

Reply:
By using a channel to collect the err returned? Or we can use panic inside goroutine, not sure if this is a good idea.

Reply:
You can simply add a variable outside the goroutine for internal use, but you might need to use a `WaitGroup` to wait for the goroutine to finish.